toffy/mph_inspect.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from ark.utils import io_utils, misc_utils
+from mibi_bin_tools.bin_files import get_histograms_per_tof
+
+
+def bin_array(arr, bin_factor):
+    arr_bin = np.cumsum(arr)
+    arr_bin[bin_factor:] = arr_bin[bin_factor:] - arr_bin[:-bin_factor]
+    arr_bin = arr_bin[bin_factor::bin_factor]
+
+    return arr_bin
+
+
+def compute_mph_intensities(bin_file_dir, fov_list, mass, mass_start, mass_stop,
```

Comment:
It's not computing the median pulse height intensities. The median is the median of the intensities. What this function does is return the actual intensity data, rather than just the median summary statistic